### DISCUSSION TITLE
Just run the wordpress demo but got this error

### DISCUSSION THREAD

**Discussion by chandamon** [vercel/next.js#16258] [Help]
Got any clue anyone? Would appreciate some pointers...thanks!

<img width="1053" alt="Screenshot 2020-08-17 at 14 39 55" src="https://user-images.githubusercontent.com/11209811/90397147-b0da4700-e097-11ea-8f55-abb8b73aeb56.png">

**Comment by jamesmosier** [comment]
What is your `WORDPRESS_API_URL` in  your `.env.local`? You'll need to setup that up [according to the README](https://github.com/vercel/next.js/tree/canary/examples/cms-wordpress#step-3-set-up-environment-variables)

**Reply by chandamon** [reply]
I did, it's http://chandamon.com/home/graphql. Or do I need to do something there too?

**Reply by jamesmosier** [reply]
You'll want to make sure to specify the url with the `https` in front: `https://www.chandamon.com/home/graphql`

**Reply by jamesmosier** [reply]
If you put `console.log(API_URL)` on line 11 (just above your error) of your `lib/api.js` file, what it the output?

**Reply by chandamon** [reply]
Forgotten to say, yes, it's https Im using.

And this is the console log

<img width="551" alt="Screenshot 2020-08-17 at 16 39 09" src="https://user-images.githubusercontent.com/11209811/90408557-454ca580-e0a8-11ea-84fb-136b4ef9216e.png">

**Reply by jamesmosier** [reply]
There should be a console log right above that error that is either null or empty (I'm guessing).

Could you replace `const res = await fetch(API_URL, {` with `const res = await fetch('https://www.chandamon.com/home/graphql', {` and see what happens?

**Reply by chandamon** [reply]
Then I get this:

<img width="1658" alt="Screenshot 2020-08-17 at 16 47 01" src="https://user-images.githubusercontent.com/11209811/90409378-5c3fc780-e0a9-11ea-9237-0a89978d2396.png">

And no, dont have any console log :-/

**Reply by jamesmosier** [reply]
1. Do you have a featured image in your post? https://github.com/vercel/next.js/tree/canary/examples/cms-wordpress#step-2-populate-content
2. If you add `console.log(posts)` above the `return (` [here](https://github.com/vercel/next.js/blob/aa7659d8078e3da30d634a8c2f88b6e9e1a5a198/examples/cms-wordpress/components/more-stories.js#L4) what is output?

**Reply by chandamon** [reply]
1. Yes I do.
2. Basically the same output, it doesnt get any console log ;-/

**Comment by abdelhak-ajbouni** [comment]
I have the same issue. @chandamon  did you fix it ?
